.buildkite/scripts/steps/test/jest_parallel.sh

```diff
@@ -13,7 +13,7 @@ exitCode=0
 while read -r config; do
   if [ "$((i % JOB_COUNT))" -eq "$JOB" ]; then
     echo "--- $ node scripts/jest --config $config"
-    node --max-old-space-size=14336 ./node_modules/.bin/jest --config="$config" --runInBand --coverage=false --passWithNoTests
```

Comment:
just curious - why this change?

Reply:
Sounds like you're on it @jbudz, but just want to be explicit: if there is a bug in scripts/jest we can fix it, but we need to revert this change.

Reply:
As was mentioned in the description, this seems to be an issue with the lack of preserve symlinks. We could add it to the Node command but I actually think the change here to use `scripts/jest` directly is probably best.

Reply:
Oh, misunderstood what was happening here and that this was starting to use `scripts/jest`... Oops!